anyway/models.py

```diff
@@ -871,13 +872,52 @@ class NewsFlash(Base):
     street2_hebrew = Column(Text(), nullable=True)
     non_urban_intersection_hebrew = Column(Text(), nullable=True)
     road_segment_name = Column(Text(), nullable=True)
+    critical = Column(Boolean(), nullable=True)
     newsflash_location_qualification = Column(
         Integer(),
         nullable=False,
         server_default=text(f"{NewsflashLocationQualification.NOT_VERIFIED.value}"),
     )
     location_qualifying_user = Column(BigInteger(), nullable=True)
 
+    def __init__(self, **kwargs):
+        super(NewsFlash, self).__init__(**kwargs)
+        self.critical = self.get_critical()
+
+    def get_critical(
+        self,
+        years_before=5,
+        suburban_road_severe_value=10,
+        suburban_road_killed_value=3,
+        urban_severe_value=2,
+    ):
+        from anyway.widgets.suburban_widgets.injured_count_by_severity_widget import (
+            InjuredCountBySeverityWidget,
+        )
+
+        if self.road1 is None or self.road_segment_name is None or self.date is None:
+            return None
```

Comment:
What happens in urban road? I think you might miss these news flashes

Reply:
how can i recognize urban road?

Reply:
both yishuv_name and street1_hebrew are not null